include/ECU.h

```diff
@@ -1,9 +1,10 @@
 #ifndef ECU_H
 #define ECU_H
 
```

Comment:
Seen this a few places - typically you would use #include <FILE> to use files you don't declare inside of the project scope (i.e. <FlexCAN_T4.h>, <BufferPacker.h>, "Throttle.h", "Brake.h"). Definitely not required, but good for consistency

Reply:
I just copy/paste boiler plate from last year that had this. unless you selected the wrong line for the comment

Reply:
It doesn't technically matter, so no harm